accepted/2022/dotnet-release-labels.md

```diff
@@ -0,0 +1,245 @@
+
+- Pre-release (previews and release candidates)
+- Full support
+- Maintenance
+- End of life/support
+
+For stable releases (post-GA), we have two support durations:
+
+- Shorter releases, supported for 18 months
+- Longer "LTS" releases, supported for 36 months
+
+The shorter releases are released in even years, and the LTS ones in odd years.
+
+## Currently-used labels
+
+Today, we have multiple labels that we apply to releases, as you can see illustrated in the following Visual Studio dialog:
+
+![image](dotnet-release-labels.png)
+
+These labels are attempting to describe the following:
+
+- .NET 5 is a "Current" or shorter-supported release.
+- .NET 6 is a "Long-term support" or longer-supported release.
+
+We've heard the feedback that "Current" is very confusing, particularly when it describes a release that isn't the latest one. That's the case in this Visual Studio image. It is easy to agree with that.
+
+## Industry examples
+
+Various projects use various terms to describe release types and support timeframes.
+
+- [Node.js](https://nodejs.org/en/about/releases/) releases have "Current", "Active LTS", and "Maintenance LTS" phases. Even-numbered releases move to these two LTS phases after the "Current" phase. Odd-numbered releases are no longer supported after the "Current" phase.
+- [OpenJDK](https://access.redhat.com/articles/1299013) releases are supported for 6 years, at least by Red Hat.
+- [Python](https://devguide.python.org/devcycle/#branches) releases have "Maintenance" and "Security" phases. Releases are supported for ~5 years.
+- [Red Hat Enterprise Linux (RHEL)](https://access.redhat.com/support/policy/updates/errata/) releases have "Full", "Maintenance", and "Extended life" support phases. Each RHEL release is LTS.
+- [Ubuntu](https://wiki.ubuntu.com/Releases) releases have "Current" and "Extended Security" support phases. Release are either regular (with no additional label) or LTS. Every two years, there are 3 regular releases and one LTS release. All currently-supported releases are considered "Current".
+
+The industry examples demonstrate that we are using the "LTS" label correctly and the "Current" label incorrectly. The Ubuntu/Canonical model is the closest to our practice.
+
+## Proposed labels
+
+New labels are proposed for a variety of purposes.
+
+**Release-type (support duration) labels:**
+
+- **STS** -- Indicates a release is supported for the Short Term Support (STS) timeframe (18 months) and currently actively supported.
+- **LTS** -- Indicates a release is supported for the Long Term Support (STS) timeframe (3 years) and currently actively supported.
+
+**Release support status labels:**
+
+- **Preview** -- Indicates a release is in preview and (unless "Go-Live") is unsupported.
+- **Go-Live** -- Indicates a preview or release candidate is supported in production.
+- **Maintenance** -- Indicates that a release is within 6 months of remaining support, with security fixes only.
```

Comment:
>with security fixes only

Does this mean "no bug fix" ?

Reply:
It means security fixes only and only the most severe of non-security fixes. This has been our policy the whole time. This isn't a change in policy. Also, I think no one has ever complained about this policy. By the time we're at the very end of support, users value stability very highly. This policy delivers that.

Reply:
Thank you for clarifying!